src/coprocessor/dag/expr/ctx.rs

```diff
@@ -186,16 +194,6 @@ impl EvalConfig {
             .set_divided_by_zero_as_warning((flags & FLAG_DIVIDED_BY_ZERO_AS_WARNING) > 0)
     }
 
-    /// detects if 'ERROR_FOR_DIVISION_BY_ZERO' mode is set in sql_mode
-    pub fn mode_error_for_division_by_zero(&self) -> bool {
-        self.sql_mode & MODE_ERROR_FOR_DIVISION_BY_ZERO == MODE_ERROR_FOR_DIVISION_BY_ZERO
-    }
-
-    /// detects if 'MODE_NO_ZERO_DATE_MODE' mode is set in sql_mode
-    pub fn mode_no_zero_date_mode(&self) -> bool {
-        self.sql_mode & MODE_NO_ZERO_DATE_MODE == MODE_NO_ZERO_DATE_MODE
-    }
-
```

Comment:
I would keep these methods, it means users of `EvalConfig` don't have to know how sql mode is implemented internally

Reply:
Thanks! I removed these functions because they simply check individual flags. It might be not DRY if we provide a function for each flag 🤨